### PR TITLE
feat: Added translations for Arabic, Bangla, Hindi, Spanish

### PR DESCRIPTION
**Pull request by RobloxianDemo**
## Description

* Added in Arabic translations;
* Fixed and finished up the Bangla translations;
* Finished up the Hindi translations;
* Finished up the Spanish translations.

### Type of change

- [x] Arabic translations;
- [x] Bangla translations;
- [x] Hindi translations;
- [x] Spanish translations.

### CODE REVIEW

File: src/translations.php [hunk 1]
```diff
@@ -34,11 +34,21 @@
         "Present" => "Present",
     ],
     // Locales below are sorted alphabetically
+    "ar" => [
+        "Total Contributions" => "إجمالي المساهمات",
```
Comment (DenverCoder1) [inline src/translations.php:38]: `"rtl" => true` can be used to mark it as a right to left language which should help with rtl/ltr run ordering

```suggestion
        "rtl" => true,
        "Total Contributions" => "إجمالي المساهمات",
```

**Review by DenverCoder1 — COMMENTED** [review]
The translations look good to me 👍

I have one comment about rtl support though

File: src/translations.php (hunk 1 above)
Comment (RobloxianDemo) [reply src/translations.php:38]: Ah, my mistake, as I hadn't noticed that previously! I'll make sure to implement that in for future translations! Thanks for the check-in!

Comment (DenverCoder1) [reply src/translations.php:38]: Feel free to use the "commit suggestion" button to co-author the fix.

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Pull request merged**